### PR TITLE
Fixed scripts for deploying and an error in creating mock db

### PR DESCRIPTION
**Pull request by clundin55**
Forgot to add database migrations for website app. Need this merged to have a clean install run.

### CODE REVIEW

File: scripts/deploy_python.sh [hunk 1]
```diff
@@ -5,6 +5,8 @@ cd ./yw_website/
 
 python manage.py makemigrations
 python manage.py migrate 
+python3 manage.py makemigrations website
+python3 manage.py migrate website
```
Comment (aniehuser) [inline scripts/deploy_python.sh]: should be just `python` for mackenzie

File: scripts/remote_deploy_python.sh [hunk 2]
```diff
@@ -7,6 +7,8 @@ pip3 install -r requirements.txt
 cd ./yw_website/
 python manage.py makemigrations
 python manage.py migrate 
+python3 manage.py makemigrations website
+python3 manage.py migrate website
```
Comment (aniehuser) [inline scripts/remote_deploy_python.sh]: same, should be just `python`

**Review by aniehuser — CHANGES_REQUESTED** [review]
was wondering what why that broke

File: scripts/deploy_python.sh (hunk 1 above)
Comment (clundin55) [reply scripts/deploy_python.sh]: fixed

File: scripts/remote_deploy_python.sh (hunk 2 above)
Comment (clundin55) [reply scripts/remote_deploy_python.sh]: fixed

**Comment by clundin55** [comment]
> was wondering what why that broke

@aniehuser The app code was separated from the website code. So the database of the app is separate so it needs it's own database migrations.

**Review by aniehuser — APPROVED** [review]

**Comment by aniehuser** [comment]
Interesting, well I tested it and it works so lets get this merged

**Pull request merged**